src/euphorie/client/settings.py

```diff
@@ -197,6 +197,15 @@ def getContent(self):
         return user
 
     def initiateRequest(self, account, login):
+        flash = IStatusMessage(self.request).addStatusMessage
+        # Make it work when acl_users is in Memcached: We need to fetch the
+        # account again, to prevent DetachedInstanceError
+        account_query = Session.query(Account).filter(Account.id == account.id)
+        if not account_query.count():
+            log.error("Account could not be fetched")
+            flash(_(u"An error occured while sending the confirmation email."), "error")
+            return False
+        account = account_query.one()
```

Comment:
I am pretty sure `Session.refresh(account)` does the same.

Reply:
Unfortunately `Session.refresh(account)` instead of the change from this PR gives me

InvalidRequestError: Instance '<Account at 0x7f94fc116a50>' is not persistent within this Session

Reply:
Ok, no problem, maybe it is inspect(account).session.refresh(account), I really do not remember by hearth

Reply:
Well, `inspect(account).session` is None in this case.